packages/murmur2/dune

```diff
@@ -1,3 +1,4 @@
 (library
  (name murmur2)
+ (libraries uutf)
```

Comment:
Do you know if it's possible to not add a dependency? Related: https://discuss.ocaml.org/t/decoding-many-unicode-strings-with-uutf/8910/2 and https://github.com/ocaml/ocaml/pull/10710.

Relevant sentence:
> And once you can afford 4.14, I suggest you ditch uutf for the [Stdlib UTF decoders](https://github.com/ocaml/ocaml/pull/10710) which do not allocate at all.

Reply:
Just remove the `uutf` adding a few lines of code. Thank you, @jchavarri